dev/tasks/verify-rc/github.nix.yml

```diff
@@ -64,8 +69,9 @@ jobs:
             fi
             if [ "$TEST_RUBY" = "1" ]; then
               ruby --version
+              sudo gem install bundler
             fi
           fi
-          # TODO: put 0.16.0 2 in some separate file?
+          # TODO: put version and rc number in some separate file?
```

Comment:
@kszucs is there currently a mechanism to pass params (or override defaults via `crossbow submit`)?

Reply:
Not yet, created jira https://issues.apache.org/jira/browse/ARROW-8525